client/src/pages/Repository/RepositoryOverview.tsx

```diff
@@ -42,8 +42,8 @@ const RepositoryOverview = ({ syncState, repository }: Props) => {
 
   const { data: readmeData, searchQuery } = useSearch<FileSearchResponse>();
   useEffect(() => {
-    const readmePath = repository.files.find((file) =>
-      file.path.includes('.md'),
+    const readmePath = repository.files.find(
+      (file) => file.path === 'README.md',
```

Comment:
I think i saw somewhere `readme.md` and `Readme.md` and `README.MD` maybe we should do case insensitive search ?